src/main/java/org/junit/runners/model/MemberValueConsumer.java

```diff
@@ -0,0 +1,8 @@
+package org.junit.runners.model;
+
+/**
+ * @since 4.13
+ */
+public interface MemberValueConsumer<T> {
```

Comment:
I like the name and the packaging. Could you add a short description of what it does in the class Javadoc, and add method Javadoc to every new public method that is in a public class?